stories/InputField.stories.js

```diff
@@ -91,4 +89,11 @@ function createStory(name, props) {
         ))
 
         .add('Required', () => <Input {...props} required />)
+
+        .add('Demo of different behavior', () => (
```

Comment:
This title is not descriptive, it should be `No value, no focus`.
But what's the difference of this one to `No placeholder, no value`?

Reply:
Oops this should not have been committed. Good catch.